bessctl/commands.py

```diff
@@ -1532,11 +1532,12 @@ def tcpdump_module(cli, module_name, direction, gate, opts):
             pass
 
 
-@cmd('track ENABLE_DISABLE [MODULE] [DIRECTION] [GATE]',
+@cmd('track ENABLE_DISABLE [DIRECTION] [MODULE] [GATE]',
      'Count the packets and batches on a gate')
```

Comment:
I agree with Panda.  Might there be another approach that achieves what you want?